k8s/pkg/apis/longhorn/v1beta2/volume.go

```diff
@@ -306,6 +306,10 @@ type VolumeSpec struct {
 	// - disabled: Disable offline rebuilding for this volume, regardless of the global setting
 	// +optional
 	OfflineRebuilding VolumeOfflineRebuilding `json:"offlineRebuilding"`
+	// RebuildingMBytesPerSecond limits the write bandwidth (in megabytes per second) on the destination replica during rebuilding.
+	// +kubebuilder:validation:Minimum=0
```

Comment:
Could you mention what `0` means in the description?

Reply:
Sure. Clarified what 0 means.

Reply:
> is it acceptable to add new commits on top of https://github.com/longhorn/longhorn-manager/commit/ec00b5a95365c114d036fba679326d82c6d0f065

This is acceptable. No worries. Thank you.